TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c

```diff
@@ -3432,11 +3432,8 @@ PUBLIC S16 nbCtxtRelSndRlsCmpl(NbUeCb *ueCb)
    relRsp.spConnId = ueCb->s1ConCb->spConnId;
    relRsp.pdu      = ctxtRelPdu;
 #ifdef MULTI_ENB_SUPPORT
-   for(enbIdx = 0; enbIdx < smCfgCb.numOfEnbs; enbIdx++)
-   {
-      relRsp.enbId = enbIdx;
-      NbIfmS1apRelRsp(&relRsp);
-   }
+   relRsp.enbId = ueCb->enbId;
```

Comment:
what's the reason to remove the for loop?

Reply:
This loop was used everywhere for getting the enb other than current enb. But it was not working as expected because we were calling the release function for every iteration without checking the actual enb id, which was sending the release from wrong ENB. Since after the handover for multi ENB scenarios, we were updating the ENB ids to the ueCB->enbId, I used that enb id directly instead of taking it from the loop.